packages/core/src/utilities/background-color/background-color.example.html

```diff
@@ -7,12 +7,14 @@
 'secondary-lightest'], ['error', 'error-dark', 'error-darkest', 'error-light', 'error-lighter',
 'error-lightest'], ['base', 'white', 'background', 'background-inverse', 'transparent'] ]; -%> <%
 groups.forEach(group => { -%>
-<section class="ds-l-row ds-u-margin-x--0 ds-u-margin-bottom--3">
+<section class="ds-u-border-bottom--1 ds-u-padding-bottom--3 ds-u-margin-bottom--4">
```

Comment:
Again, I did not document or communicate this anywhere haha, but I made some classes for common use cases. This way we have have consistent border and spacing, semantic markup in our examples, and also solve common issues like having a border on the last section.

In this case you can use `example--list` 
```
<section class="example--list">
  <% var groups = [ ['primary', 'primary-darker', 'primary-darkest'], ['primary-alt',
    'primary-alt-dark', 'primary-alt-darkest', 'primary-alt-light', 'primary-alt-lightest'], ['gray',
    'gray-dark', 'gray-light', 'gray-lighter', 'gray-lightest'], ['gold', 'gold-light', 'gold-lighter',
    'gold-lightest'], ['warn', 'warn-light', 'warn-lighter', 'warn-lightest'], ['green', 'green-light',
    'green-lighter', 'green-lightest'], ['success', 'success-light', 'success-lighter',
    'success-lightest'], ['secondary', 'secondary-dark', 'secondary-darkest', 'secondary-light',
    'secondary-lightest'], ['error', 'error-dark', 'error-darkest', 'error-light', 'error-lighter',
    'error-lightest'], ['base', 'white', 'background', 'background-inverse', 'transparent'] ]; -%> <%
    groups.forEach(group => { -%>
  <article>
    <% group.forEach(color => { -%>
    <div class="ds-u-margin-y--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--transparent">.ds-u-fill--<%= color %></code>
      <code class="c-swatch__label js-swatch-hex ds-u-fill--transparent ds-u-color--gray">#</code>
    </div>
    <% }) -%>
  </article>
  <% }) %>
</section>
```
Here's what it would look like: https://gyazo.com/4f3169d7d938e5cd79effcf95d58dc21